packages/ember-application/lib/system/resolver.js

```diff
@@ -117,11 +117,9 @@ export default EmberObject.extend({
   init() {
     this._parseNameCache = dictionary(null);
   },
+
   normalize(fullName) {
-    let [
-      type,
-      name
-    ] = fullName.split(':', 2);
+    let [ type, name ] = fullName.split(':');
```

Comment:
also split `[ type, name ]` (unless loose mode is enabled for babel) produced some gnarly output.

If we are not using loose, lets move away from array destructuring, if we are we can leave it.